src/strategies/base.ts

```diff
@@ -148,6 +150,7 @@ export abstract class BaseStrategy implements Strategy {
     this.extraFiles = options.extraFiles || [];
     this.initialVersion = options.initialVersion;
     this.extraLabels = options.extraLabels || [];
+    this.dateFormat = options.dateFormat || '%Y-%m-%d';
```

Comment:
nit: define as a constant like `DEFAULT_DATE_FORMAT` in this file

Reply:
I set this as a constant in `src/updaters/generic.ts` and imported it from there.